### DISCUSSION TITLE
urun exploits

### DISCUSSION THREAD

**Discussion by ohiobombs** [3kh0/ext-remover#1080] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What are some Urun exploits that i could try?

**Comment by Blobby-Boi** [comment]
wdym urun exploits

**Reply by ohiobombs** [reply]
like some bookmarklets that work with Urun

**Reply by ohiobombs** [reply]
the ublock filter javascript runner thing (idk if u were unsure about that or something)

**Reply by AshtonDavies** [reply, marked as answer]
Nothing universal that is an exploit can be done with it. Extensions can only access web and file pages.

**Reply by ohiobombs** [reply]
alr
